### ISSUE TITLE
Question about dataset

### ISSUE THREAD

**Issue by kindra130801** [YUCHEN005/NASE#5]
Hello, let me ask about dataset. Is the audio in the noisy folder pure noisy audio or mixed audio with clean audio and noisy audio?

**Comment by YUCHEN005** [comment]
Hi, the noisy folder is actually the vb-demand test set, you can check the noise info of each wav file at: https://github.com/YUCHEN005/NASE/blob/master/noise_labels/log_testset.txt

**Comment by kindra130801** [comment]
Is it the same in the training dataset? Thank you!

**Comment by YUCHEN005** [comment]
Both train and test datasets have separate clean/noisy versions, you can find the details at https://datashare.ed.ac.uk/handle/10283/2791
